src/GraphDrawer/GraphDrawer.js

```diff
@@ -22,7 +22,7 @@ class GraphDrawer extends Component {
 
   getData = (uuid, radius) => {
 
-    return fetch("https://lrwu47ypal.execute-api.us-east-1.amazonaws.com/dev/metrics", {
+    return fetch("https://m0mgf48bn4.execute-api.us-east-1.amazonaws.com/dev/metrics", {
```

Comment:
This is a bug - the endpoint was still pointing to the old API. It's now updated to point to the new one